scripts/download-unicode-files.sh

```diff
@@ -30,7 +30,7 @@ done
 
 for filename in $emoji_files ; do
```

Comment:
Wouldn't filenames be split here, in `$emoji_files`, anyway?

Reply:
Tell that to shellcheck :)